src/main/java/ai/elimu/model/enums/Language.java

```diff
@@ -11,8 +11,8 @@
     HIN("hin", "Hindi", "हिन्दी"),
     SWA("swa", "Swahili", "Kiswahili"),
     URD("urd", "Urdu", "اردو"),
-    XHO("xho", "Xhosa", "isiXhosa");
-    
+    XHO("xho", "Xhosa", "isiXhosa"),
+    HAU("hau", "Hausa", "Hausa");
```

Comment:
@usfarouk How do natives refer to the Hausa language? Do they say "Harshen Hausa"?

Reply:
Its referred to as "harshen hausa"

Reply:
@nya-elimuai yes, they do. 
@ahmadafreq thank you

Reply:
@usfarouk Ok. I was asking because at https://en.wikipedia.org/wiki/List_of_ISO_639-2_codes I read "Harshen Hausa; هَرْشَن".

Can you update this pull request with the correct value for the `nativeName` attribute?